aten/src/ATen/native/cuda/TensorModeKernel.cuh

```diff
@@ -4,6 +4,8 @@
 #include <ATen/native/cuda/Loops.cuh>
 #include <ATen/native/cuda/SortingCommon.cuh>
 
+#include <THC/THCReduce.cuh>
```

Comment:
is this include needed?

Reply:
Yes, it was previously included indirectly from `THCTensorMathReduce.cuh` but I've removed the include from that header.